sw/host/provisioning/ft_lib/src/lib.rs

```diff
@@ -86,10 +87,17 @@ pub fn run_sram_ft_individualize(
     reset_delay: Duration,
     sram_program: &SramProgramParams,
     ft_individualize_data_in: &ManufFtIndividualizeData,
-    spi_console: &SpiConsoleDevice,
+    console_spi: &String,
     timeout: Duration,
     ujson_payloads: &mut UjsonPayloads,
 ) -> Result<()> {
+    // Setup the SPI console with the GPIO TX indicator pin.
+    let spi = transport.spi(console_spi)?;
+    let device_console_tx_ready_pin = &transport.gpio_pin("IOA5")?;
```

Comment:
This should really be a parameter the ultimately comes from command line args.  Ok to fix in a follow-up PR.

Reply:
fixed